charts/ccsm-helm/charts/identity/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+description: Identity Helm Chart for Kubernetes
+name: identity
+version: 0.0.25
+type: application
+icon: https://raw.githubusercontent.com/camunda-cloud/camunda-cloud-documentation/master/static/img/camunda-operate-gradient.png
+annotations:
+  artifacthub.io/changes: |
+    - create identity sub chart
+  artifacthub.io/containsSecurityUpdates: "false"
+dependencies:
+  - name: keycloak
+    repository: "https://charts.bitnami.com/bitnami"
+    version: 7.1.6
+    condition: "identity.enabled"
+  - name: common
```

Comment:
:question: Isn't the common also a dependency of [keycloak](https://github.com/bitnami/charts/blob/master/bitnami/keycloak/Chart.yaml#L10-L14)?  Do we have to specify it again? Or is it because you use it directly in `charts/ccsm-helm/charts/identity/templates/deployment.yaml`? Sorry not an Helm expert.

Reply:
Actually the condition is not really necessary here, since the complete subchart is excluded already in the parent if the condition is false.

We need the commons for the templates we use correct, without it didnt work.

Reply:
I removed both. Whereas I'm bit unsure about the dependency, since I think being explicit is also good.